src/AppHeader.js

```diff
@@ -0,0 +1,24 @@
+function AppHeader({ inputChange, addItem, inputValue }) {
+    return (
+        <div className='header'>
+            <h2 className='header__title'>To Do List</h2>
+            <div className='header__input'>
+                <input
+                    className='input'
+                    type='text'
+                    id='input'
+                    placeholder='New Task...'
+                    onChange={inputChange}
+                    value={inputValue}
+                />
+                <span
```

Comment:
in your case, it's behave like a button, but you use `span` here, this is wrong from semantic HTML perpective

Reply:
Sure, I changed it to `button`.